app/views/claims/support/claims/_secondary_navigation.html.erb

```diff
@@ -0,0 +1,8 @@
+<% unless HostingEnvironment.env.production? %>
```

Comment:
Have you considered using a feature flag instead of this environment check? I'm assuming this feature is actually something you want enabled at some point.

Reply:
I did consider it, but it seemed like more effort to use a feature flag in the case of where we only want to prevent it from being accessible in production, not in any of the pre-production environments.

Reply:
Whilst it might be more effort this is the very purpose for feature flags.
It means that these 3 features can be developed and released independently of each other.
Otherwise, you'll need to complete the work for all 3 features before removing this `HostingEnvironment.env.production?` clause.

In my opinion, `HostingEnvironment.env.production?` should only be used if this feature is intended to never be used on production. Such as your database cleanup work.

Reply:
👍 I'll make the change.

Reply:
I've gone ahead with using an ENV var as a feature flag to get around Flipflop's limitation of requiring a database connection.

Reply:
Whilst I understand your change to use ENV variables instead of feature flags. I would have expected a flag per feature (Sampling, Clawback, etc.). 
Is the intention onto to release these features to production only once all 3 features are complete?

Reply:
Yes. The intention is to release all of these features all together.

That being said, our recent revisions have broken down the processes into distinct stages.

I'll remove Sampling and Clawback for now.